net-misc/insync/insync-3.8.4.50481-r2.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 1999-2023 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit unpacker xdg
+
+DESCRIPTION="Advanced cross-platform Dropbox, Google Drive and Microsoft OneDrive client"
+HOMEPAGE="https://www.insynchq.com/"
+
+SRC_URI="https://cdn.insynchq.com/builds/linux/insync_${PV}-bullseye_amd64.deb"
+
+RESTRICT="strip"
+
+LICENSE="as-is"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="wayland kde xfce"
+
+RDEPEND="
+	app-crypt/gnupg
+	dev-libs/libthai
+	dev-libs/nss
+	dev-qt/qtvirtualkeyboard
+	sys-devel/gcc
+	>=sys-libs/glibc-2.31
+	x11-misc/xdg-utils
+	wayland? ( dev-libs/wayland )
```

Comment:
This is an arbitrary change, please restore it to
```
	wayland? (
		dev-libs/wayland
	)
```
to keep the better readability.